src/data/set/finite.lean

```diff
@@ -994,6 +994,33 @@ by { rw ←range_subset_iff at hf, exact (infinite_range_of_injective hi).mono h
 lemma infinite.exists_nat_lt {s : set ℕ} (hs : s.infinite) (n : ℕ) : ∃ m ∈ s, n < m :=
 let ⟨m, hm⟩ := (hs.diff $ set.finite_le_nat n).nonempty in ⟨m, by simpa using hm⟩
 
+lemma arb_large_infinite {S : set ℕ} : S.infinite → ∀ (n : ℕ ), ∃ (m : ℕ), (m ∈ S ∧ m > n) :=
+begin
+  intros h n,
+  apply bex_def.mp,
+  apply set.infinite.exists_nat_lt h,
+end
+
+lemma infinite_exists_nat_lt {S : set ℕ} : (∀ (n : ℕ ), ∃ (m : ℕ), (m ∈ S ∧ m > n)) → S.infinite :=
```

Comment:
I think this would be a better name and statement
```suggestion
lemma infinite_of_forall_exists_nat_lt {S : set ℕ} (h : ∀ n, ∃ m ∈ s, n < m) : S.infinite :=
```